%----------------------------------------------------------------------------------------
%	LEFT COLUMN
%----------------------------------------------------------------------------------------

\begin{minipage}[t]{0.33\textwidth} % The left column takes up 33% of the text width of the page
%------------------------------------------------
% Education
%------------------------------------------------

\section{Education} 
\runsubsection{BA}
\descript{| Biological Sciences}
\location{George Washington University | 2017}

\vspace{\topsep} % Hacky fix for awkward extra vertical space
\vspace{1pt}
\begin{tightitemize}
\item Concentration in Bioinformatics
\item Cum Laude
\item GPA: 3.51/4.0
\end{tightitemize}
\vspace{3pt}

%------------------------------------------------
% Skills
%------------------------------------------------
\section{Skills}

\descript{Computing}
\location{Programming Languages}
Python \textbullet{} R \textbullet{} Bash \\ 
Nextflow \textbullet{} CWL \\
\location{Operating Systems}
Linux \textbullet{} Mac \textbullet{} Windows\\
\location{HPC Platforms}
Slurm \textbullet{} LSF \textbullet{} SGE \textbullet{} AWS \\

\location{IDEs \& Reproducible Research}
Jupyter \textbullet{} RStudio \textbullet{} R Shiny\\
Vim \textbullet{} Docker \textbullet{} Singularity \\
\sectionspace % Some whitespace after the section

\descript{Bioinformatics}
\location{Data Types}
Whole Exome Sequencing (WES) \\
RNA-seq \\

\location{Statistics}
Regression \textbullet{} Survival Analysis \textbullet{} \\
Dimmensionality Reduction \textbullet{} 
Clustering \textbullet{} \\
Machine Learning Models \\

\location{Repositories \& Databases}
TCGA \textbullet{} MMRF-COMMPASS \textbullet{} \\
GEO/SRA \textbullet{} GitHub \textbullet{} Bioconductor \\

\location{Sequence Alignment}
BWA \textbullet{} STAR \textbullet{} Bowtie \\

\location{SNV and CNV calling}
GATK \textbullet{} Samtools \\
VarScan2 \textbullet{} CNVkit \textbullet{} XHMM \\

\location{Differential Gene Expression}
limma/voom \textbullet{} DEseq2 \textbullet{} Cufflinks \\

\location{Immunogenomics}
Immune Repertoire Analysis \textbullet{} \\
in-silico Neo-antigen Prediction \\
\sectionspace % Some whitespace after the section

\descript{General}
\location{Software}
\LaTeX \textbullet{} Photoshop \textbullet{} Illustrator \\
\location{Languages}
English (Native) \textbullet{} Spanish (Fluent)

\sectionspace % Some whitespace after the section

\end{minipage} % The end of the left column
\hfill
%
%----------------------------------------------------------------------------------------
%	RIGHT COLUMN
%----------------------------------------------------------------------------------------
%
\begin{minipage}[t]{0.66\textwidth} % The right column takes up 66% of the text width of the page

%------------------------------------------------
% Experience
%------------------------------------------------

\section{Experience}


\runsubsection{Icahn School of Medicine}
\descript{| Associate Researcher II (Bioinformatics)}
\location{\textbf{Department of Medical Oncology \& Hematology}}
\location{Lab of Dr. Samir Parekh | Sep 2020 - Present}
\vspace{\topsep} % Hacky fix for awkward extra vertical space
\begin{tightitemize}
\item Led project on identifying a transcriptomic predictive gene signature for drug response to selinexor in multiple myeloma using RNA-seq from multiple clinical trial cohorts.
\item Contributed to application development of various tools as part of an in-house clinical precision medicine pipeline for multiple myeloma. Processed and analysed RNA-seq and WES data for samples as part of precision medicine pipeline.
\item Designed and developed the implementation of reports used in the mutliple myeloma precision medicine pipeline at Mount Sinai.
\item Collaborated on multiple projects with experimental wet-bench biologists, with clinicians, and with external biotech industry partners, providing data science support as needed. 
\item Presented research findings at internal department seminars and externally at conference poster sessions. Drafted and prepared manuscripts for submission to peer-reviewed journals.
\end{tightitemize}

\sectionspace % Some whitespace after the section

%------------------------------------------------


\runsubsection{Icahn School of Medicine}
\descript{| Associate Researcher I (Bioinformatics)}
\location{\textbf{Department of Genetics \& Genomics Sciences}}
\location{Lab of Dr. Bojan Losic | May 2018 - Sep 2020}
\begin{tightitemize}
\item Conducted research in multiple disease systems, including inflammatory bowel disease (IBD), glioblastoma, and hepatocellular carcinoma.
\item Integrated multi-omics data, including RNA-seq, whole-exome sequencing (WES), and RRBS methylation profiling to characterize intra-tumoral heterogeneity and its relationship to the tumor micro-environment.
\item Built WES data processing pipelines for internal use using Nextflow. Managed data storage and organization for a large multi-omics dataset in collaboration with biotech industry partners using ad-hoc scripts.
\item Collaborated on multiple projects with other departments internally, with clinicians, and with external biotech industry partners.
\item Presented research findings at internal department seminars and externally at conference poster sessions. Drafted and prepared two manuscripts for submission to peer-reviewed journals.
\end{tightitemize}

\sectionspace % Some whitespace after the section

%------------------------------------------------

\runsubsection{George Washington University}
\descript{| Undergraduate Research Assistant}
\location{\textbf{Department of Pharmacology \& Physiology}}
\location{Lab of Dr. Anelia Horvath | Aug 2015 - May 2017}
\begin{tightitemize}
\item Integrated WES and RNA-seq data from TCGA to characterize the gene expression profiles of tumor suppressor and oncogenes in cancer.
\item Processed WES and RNA-seq data using standard pipelines for alignment, QC, and variant calling. Wrote and implemented ad-hoc filtering and processing scripts using Python.
\item Contributed to manuscript preparation and data analysis for three peer-reviewed publications.
\item Trained nine new lab members in using relevant pipelines, software, and analysis. 
\end{tightitemize}
\sectionspace % Some whitespace after the section

%----------------------------------------------------------------------
\tikz[remember picture,overlay] \node[xshift=-3.25cm, yshift=2cm, opacity=0.3] at (current page.south east){\includegraphics[width=1in,height=1in]{icon.png}};

\end{minipage} % The end of the right column
\vspace*{\fill}
\center{\textcolor{gray}{1/2}}